### DISCUSSION TITLE
Out in the wild / public examples

### DISCUSSION THREAD

**Discussion by sgilligan** [duckdb/duckdb-wasm#1359] [Show and tell]
Hi All,

I thought I'd share a public example of duckdb-wasm in use, which is: https://galh.eu/  (for context: https://www.rmit.eu/galh). We use duckdb wasm in-browser for a number of projects, this atlas is our most recent. This project doesn't leverage huge volumes of data, but the dynamic query capability is the key enabler. The underlying data is static, and pre-processed into an arrow IPC store which is loaded at startup. Thank-you for this great tech! 

Simon

**Comment by carlopi** [comment]
Thanks for sharing!

That makes me think that it would be nice to have example of usage somewhere, I opened an issue about it here: https://github.com/duckdb/duckdb-web/issues/957.

**Comment by carstonhernke** [comment]
Hi @sgilligan, thank you for creating this thread! I love the data selection + map view side-by-side. Great performance and feels very snappy as well (expect nothing less from DuckDB 😁 ) 

Here's another example I've been working on which you may be interested in: [https://app.honeycomb.place/](https://app.honeycomb.place/). It uses DuckDB's support for parquet range queries to allow you to easily find the population at any place in the world.

**Reply by sgilligan** [reply]
@carstonhernke nice work with Honeycomb :) We haven't worked with range queries yet, so good to see it in action.
